libs/api/domains/auth/src/lib/models/scopeTreeNode.model.ts

```diff
@@ -0,0 +1,8 @@
+import { createUnionType } from '@nestjs/graphql'
+import { ApiScopeGroup } from './apiScopeGroup.model'
+import { ApiScope } from './apiScope.model'
+
+export const ScopeTreeNode = createUnionType({
+  name: 'AuthScopeTreeNode',
+  types: () => [ApiScope, ApiScopeGroup],
```

Comment:
Yeah I started with `as const` like in the docs, but TypeScript doesn't allow it for some reason...

![image](https://user-images.githubusercontent.com/115094/195814362-a5f56005-e876-4947-9429-d28fdf2edd88.png)

Might be an issue with our TypeScript or NestJS version. Haven't experienced any problems with this so far. Maybe since we're not using this between projects.

Reply:
Yes seams to be versioning problem
https://github.com/nestjs/graphql/issues/2122